src/metabase/driver/postgres.clj

```diff
@@ -701,6 +701,8 @@
                 (sql-jdbc.common/handle-additional-options it details-map))]
     props))
 
+(defmethod sql-jdbc.sync/excluded-schemas :postgres [_driver] #{"information_schema" "pg_catalog"})
```

Comment:
`sql-jdbc.sync/excluded-schemas` was missing an implementation for postgres, but it's already implemented for mysql and h2.